Renderer.js

```diff
@@ -0,0 +1,30 @@
+
+class Renderer {
+    constructor() {
+
+    }
+    renderUser(data) {
```

Comment:
why are you not using handlebars? You will need to use handlebars to pass the exam next week